.circleci/config.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           command: yarn install
       - run:
           name: Lint Sass and JS
```

Comment:
I think the rescription of this runner should match the name of the `yarn` endpoint. If its name in CircleCI is "lint", then `yarn lint` makes much more sense. If you're going to use `yarn test`, then this CI endpoint should be differently described.

Ideally `./run test` or `yarn test` should be running some tests, and even if it's not right now, we should describe it as a test task, and then someone can trivially come along later and add some tests.

Reply:
Done

Reply:
I was actually referring to the `name`, "Lint Sass and JS" does not necessarily describe `yarn test`.

In fact my recommendation would be that you keep the ID of this task as `lint`, keep the description as it is or change it to simply "Lint code files", and leave the command as `yarn lint` - i.e. change nothing. This is because it's nice when submitting a PR to immediately see that the linter specifically failed, rather than that tests in general failed. So in most CI configs, we have separate tasks for `lint` and `test`. And if you have no tests, you might as well just provide `lint` for now.